element_facemap/facial_behavior_estimation.py

```diff
@@ -230,20 +233,29 @@ def make(self, key):
 
         if task_mode == 'trigger':
             from facemap.process import run as facemap_run
-            facemap_params, motSVD, movSVD = (FacemapTask & key).fetch1('facemap_params', 'do_mot_svd', 'do_mov_svd')
+            params = (FacemapTask & key).fetch1('facemap_params')
+
+            # motSVD is initially a boolean and then is overwritten with a combination
+            # of lists and arrays
+            if (params['motSVD'] is True) or (len(params['motSVD']) > 0):
+                do_motSVD = True
+            else:
+                do_motSVD = False
+
+            if (params['movSVD'] is True) or (len(params['movSVD']) > 0):
+                do_movSVD = True
+            else:
+                do_movSVD = False
```

Comment:
```suggestion
            do_movSVD = bool(params['movSVD'])
```

Reply:
In the event that `params['movSVD']` is empty (e.g., `np.array([])`), this will throw a warning for numpy 1.21
```
bool(np.array([]))
<ipython-input-5-b38e717764f1>:1: DeprecationWarning: The truth value of an empty array is ambiguous.
Returning False, but in future this will result in an error. Use `array.size > 0` to check that an array is not empty.
```

Reply:
Found it! It is `any(params['movSVD'])`

Reply:
Looks like `any` will also be depreciated. Let's keep it this way for now...

Reply:
This part is outdated after the most recent changes.